modules/misunderstood/src/hooks/after_incoming_middleware/00_misunderstood_flag.js

```diff
@@ -1,7 +1,27 @@
 const axios = require('axios')
+const _ = require('lodash')
 
 const flag = async () => {
-  if (event.type === 'text' && event.nlu.intent.name === 'none') {
+  const matchesQuickReply = async () => {
+    const { currentFlow } = event.state.context
+    if (currentFlow && currentFlow.startsWith('skills/choice')) {
+      const flow = await bp.ghost.forBot(event.botId).readFileAsObject('flows', currentFlow)
+      const preview = event.preview.toLowerCase()
+      const kw = _.chain(flow.skillData.keywords)
+        .values()
+        .flatten()
+        .find(kw => kw.toLowerCase() === preview.toLowerCase())
+        .values()
```

Comment:
Still not correct. `values()` returns the values of an object, while `.value()` returns the unwrapped object. Can you test it after making your changes to make sure it works as expected?

Reply:
Oops good catch, classic case of committing a quick fix without re-testing 😅 
Now works using 0661d2e